lib/easypost/client.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require_relative 'services'
+require_relative 'http_client'
+require 'json'
+
+class EasyPost::Client
```

Comment:
praise: was going to ask about making some of this private until I realized you already did!